spec/factories.rb

```diff
@@ -264,4 +264,12 @@
     first_name { Faker::Name.first_name }
     last_name { Faker::Name.last_name }
   end
+
+  factory :provider_users_provider do
+    provider
+    provider_user
+
+    provider_id { provider.id }
+    provider_user_id { provider_user.id }
```

Comment:
These may be not necessary, the model should have the attributes.

Reply:
You are right!